Snakefile

```diff
@@ -203,3 +211,27 @@ rule bam_to_fastq:
         " I={input.bam}"
         " FASTQ={output.r1_fastq}"
         " SECOND_END_FASTQ={output.r2_fastq} 2>> {log}"
+
+if config['call_variants']:
```

Comment:
Personally I would default to calling variants since if no reference file is supplied I think it is safe to assume the user wants to call variants on the sample. To encourage users to supply a reference when available, a basic usage or best practices example can be provided in the README.

Then the structure would roughly follow:
```
if config[reference]:
    <do things>
else:
    <call_variants>
```

Reply:
Yep, this is definitly simpler than the current logic, will change!